lib/origin/selector.rb

```diff
@@ -168,7 +171,24 @@ def evolve_hash(serializer, value)
     #
     # @since 1.0.0
     def multi_selection?(key)
-      key =~ /\$and|\$or/
+      key =~ /\$and|\$or|\$nor/
+    end
+
+    # Determines if the selection operator is $in or $nin
+    # selection.
+    #
+    # @api private
+    #
+    # @example Is the selection operator $nin?
+    #   selector.multi_variable??("$nin")
+    #
+    # @param [ String ] key The key to check.
+    #
+    # @return [ true, false ] If the key is $in or $nin.
+    #
+    # @since 1.0.0
```

Comment:
The @ since version should be the next release version. Looking at https://github.com/mongoid/origin/blob/master/lib/origin/version.rb, the next release version would be 2.1.1